Header header

uint32 serial_number
uint32 handle
string model_name
uint16 model
uint32 part_number
uint32 firmware_version
int64 power_up_timestamp
int64 time_sync_offset
int64 time_sync_drift
uint8 return_count
uint8 channel_count
uint32 status_flags
uint16 temperature
uint32 fault_summary
uint8[32] fault_entries